droidlet/lowlevel/hello_robot/remote/remote_hello_robot.py

```diff
@@ -233,10 +236,23 @@ def go_to_relative(self, xyt_position):
             def obstacle_fn():
                 return self.cam.is_obstacle_in_front()
 
-            status = goto(self._robot, list(xyt_position), dryrun=False, obstacle_fn=obstacle_fn)
+            status = goto(self, list(xyt_position), dryrun=False, obstacle_fn=obstacle_fn)
             self._done = True
         return status
 
+    def is_base_moving(self):
+        robot = self._robot
+        left_wheel_moving = (
+            robot.base.left_wheel.status["is_moving_filtered"]
+            or robot.base.left_wheel.status["is_moving"]
+        )
+        right_wheel_moving = (
+            robot.base.right_wheel.status["is_moving_filtered"]
+            or robot.base.right_wheel.status["is_moving"]
+        )
+        is_moving = left_wheel_moving or right_wheel_moving
+        return is_moving
+
     def is_busy(self):
```

Comment:
Should this include arm motion?

Reply:
yes, but after we implement the arm capabilities